modules/transactions/src/main/java/org/apache/ignite/internal/tx/configuration/TransactionConfigurationSchema.java

```diff
@@ -41,10 +41,29 @@ public class TransactionConfigurationSchema {
     @Value(hasDefault = true)
     public final long timeout = 10_000;
 
-    /** Timeout for implicit transactions (milliseconds). */
     @Range(min = 1)
     @Value(hasDefault = true)
-    public final long implicitTransactionTimeout = 3_000;
+    public final long minRoTimeout = 1;
```

Comment:
1. Javadocs are missing
2. Abbreviations like RO and RW look ugly in user-facing API. Should we name them `readOnlyXXX` and `readWriteXXX`?

Reply:
done